test/notBoth.js

```diff
@@ -0,0 +1,47 @@
+import S from 'sanctuary';
+
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+describe('notBoth', function() {
+  it('combines two boolean-returning functions into one', function() {
+    const even = function(x) { return x % 2 === 0 };
+    const gt10 = function(x) { return x > 10 };
+    const f = RA.notBoth(even, gt10);
+    eq(f(8), true);
+    eq(f(13), true);
+    eq(f(9), true);
+    eq(f(14), false);
+  });
+
+  it('accepts functions that take multiple parameters', function() {
+    const between = function(a, b, c) { return a < b && b < c };
+    const total20 = function(a, b, c) { return a + b + c === 20 };
+    const f = RA.notBoth(between, total20);
+    eq(f(4, 5, 11), false);
+    eq(f(12, 2, 7), true);
+    eq(f(12, 2, 6), true);
+    eq(f(5, 6, 15), true);
+  });
+
+  it('does not evaluate the second expression if the first one is false', function() {
+    let effect = 'not evaluated';
+    const F = function() { return false };
+    const Z = function() { effect = 'Z got evaluated' };
+    RA.notBoth(F, Z)();
+    eq(effect, 'not evaluated');
+  });
+
+  it('accepts fantasy-land applicative functors', function() {
+    const { Just, Nothing } = S;
```

Comment:
More optimal would be to do that as import statement
```js
import { Just, Nothing } from S;
```
But anyway ramda-adjunct is already using monadic types from monet librarie, and we are aiming to have our own in the future (see [here](https://github.com/char0n/ramda-adjunct/blob/e9260f67a44954d0c3ca350483fb0b1a9bab49ff/src/fantasy-land/Identity.js#L16)). No need to use sanctuary for now.

See here how we already utilize [monet](https://github.com/char0n/ramda-adjunct/blob/baa62ea258deb95ed85b83cbc2a9c8a1722fbdc8/test/liftF.js#L7) for our tests.